apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -494,7 +497,7 @@ get_top_empty_chain(_Config) ->
     {ok, GenBlock} = aehttp_api_parser:decode(block, GenBlockMap),
     ExpectedMap = header_to_endpoint_top(aec_blocks:to_header(GenBlock)),
     ct:log("Cleaned top header = ~p", [ExpectedMap]),
-    HeaderMap = ExpectedMap,
+    {ExpectedMap, _} = {HeaderMap, {expected, ExpectedMap}},
```

Comment:
Why not `-include_lib("stdlib/include/assert.hrl").` and then use `?assertEqual(ExpectedMap, HeaderMap),`instead.

Reply:
Not including an extra include file though made no sense. Addressed in 8d3d441a